chart/templates/rbac/pod-launcher-rolebinding.yaml

```diff
@@ -19,8 +19,7 @@
 ## Airflow Pod Launcher Role Binding
 #################################
 {{- if and .Values.rbacEnabled .Values.allowPodLaunching }}
-{{- $schedulerLaunchExecutors := list "LocalExecutor" "SequentialExecutor" "KubernetesExecutor" "CeleryKubernetesExecutor" }}
-{{- $workerLaunchExecutors := list "CeleryExecutor" "KubernetesExecutor" "CeleryKubernetesExecutor" }}
+{{- $podLaunchExecutors := list "LocalExecutor" "KubernetesExecutor" "CeleryKubernetesExecutor" }}
```

Comment:
I may have missed or misunderstood something: may you please help clarify why `CeleryExecutor` is not in this list?

Reply:
Thanks. This is a mistake. I have fixed it and updated the documentation.